packages/i18n/src/locales/en.i18n.json

```
@@ -3773,7 +3775,8 @@
   "Message_VideoRecorderEnabledDescription": "Requires 'video/webm' files to be an accepted media type within 'File Upload' settings.",
   "messages": "messages",
   "Messages": "Messages",
-  "Messages_selected": "Messages selected",
+  "number_messages_selected": "{{count}} messages selected",
```

```suggestion
  "__count__messages_selected": "{{count}} messages selected",
```

Not sure if this is still enforced today, but we have a pattern of adding double underscores before and after a property, in the key, like this:  `__property__`.

Also, the property referenced in the key usually has the same name as the property used inside the brackets. Ex: `__count__messages_selected` instead of `number`